tools/who_what_benchmark/README.md

```diff
@@ -11,8 +16,43 @@
 * Validation of text-to-image pipelines. Computes similarity score between generated images:
     * Supports Diffusers library and Optimum-Intel via `Text2ImageEvaluator` class.
 
-The main idea is to compare similarity of text generation between baseline and optimized LLMs.
+### Installation
+Install WWB and its requirements from the source using `pip` or any other package manager. For example,
+
+* `python -m venv eval_env`
+* `source eval_env/bin/activate`
+* `pip install -r requirements.txt`
+* `pip install .`
+
+## Usage
+### Compare Text-generation Models (LLMs)
+```sh
+# Collect ground truth from the baseline Hugging Face Transformer model 
+wwb --base-model microsoft/Phi-3-mini-4k-instruct --gt-data gt.csv --model-type text --hf
+
+# Convert model to Optimum-Intel (quantized to 8-bit by default)
+optimum-cli export openvino -m microsoft/Phi-3-mini-4k-instruct phi-3-openvino
 
+# Measure similarity metric for Optimum-OpenVINO inference backend
+wwb --target-model phi-3-openvino --gt-data gt.csv --model-type text
+
+# Measure similarity metric for OpenVINO GenAI inference backend
+wwb --target-model phi-3-openvino --gt-data gt.csv --model-type text --genai
+```
+
+### Compare Text-to-image models (Diffusers)
+```sh
+# Export FP16 model to OpenVINO
+optimum-cli export openvino -m SimianLuo/LCM_Dreamshaper_v7 --weight-format fp16 sd-lcm-fp16
+# Export model with 8-bit quantized weights to OpenVINO
+optimum-cli export openvino -m SimianLuo/LCM_Dreamshaper_v7 --weight-format int8 sd-lcm-int8
+# Collect the references
+wwb --base-model sd-lcm-fp16 --gt-data lcm_test/sd_xl.json --model-type text-to-image
```

Comment:
running with --gt-data argument slightly confusing me, as user who not yet looking into help yet (of course I read that, just imagine the situation), I may suppose that it is some file that required to be manually prepared or store dataset input entries. Maybe it make sense to a little bit extend comment that reference data will be saved in this file?

P.S. I also found that there is no mention json for text2image case support in helper string and ther is small typo here 

https://github.com/openvinotoolkit/openvino.genai/blob/master/tools/who_what_benchmark/whowhatbench/wwb.py#L219

I defined and not exists -> If defined and not exists

Reply:
Also I already mentioned that in chat, if argument not provided, tool will file with not very clear error message

```
  File "/home/ea/work/py311/lib/python3.11/site-packages/whowhatbench/text2image_evaluator.py", line 59, in __init__

    self.gt_dir = os.path.dirname(gt_data)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 152, in dirname
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

is it possible to handle case and ask to provide instead?

Reply:
ok, from what I got you are proposing:
- Extend the comment and make it clear why .json file is used
- Verify that the path specified by the argument exists

right?

Reply:
> Verify that the path specified by the argument exists

no, I mean not verify that path exists (in base line running it is expected that it may not exists), but verify that argument provided, the default argument is None now if user run command like:

```
wwb --base-model SimianLuo/LCM_Dreamshaper_v7 --model-type text-to-image
```
i

Reply:
got you. Thanks for discovering this.

Reply:
fixed both comments. Added meaningful messages instead of asserts.